modules/govuk_env_sync/files/govuk_env_sync.sh

```diff
@@ -340,10 +340,19 @@ function restore_elasticsearch {
 }
 
 function dump_postgresql {
+  pg_server_version=$(psql -U aws_db_admin -h "${db_hostname}" --no-password postgres -c 'show server_version;' | sed -n '3p')
+
+  # We do this to cleanup the variable as it doesn't pass the sanity check otherwise
+  # There is probably a better way of calling sed to avoid this
+  pg_server_version=$(echo $pg_server_version)
+
+  # Test if we have what look like a postgresql version number
+  [[ $pg_server_version =~ ^[0-9]{1,2}\.[0-9]{1,2}(\.[0-9]{1,2})?$ ]] || { echo "$pg_server_version doesn't look like a proper Postgresql version" ; exit 1; }
+
   if [ -e "/etc/facter/facts.d/aws_environment.txt" ]; then
     # We do not need sudo rights to write the output file
     # shellcheck disable=SC2024
```

Comment:
We can get rid of this comment and shellcheck pragma now that we're doing `-f /tmp/${filename}` (assuming it's okay to write the file as root).